src/util/tools.ts

```diff
@@ -305,8 +305,8 @@ export const getMergePreview = (
   }
 }
 
-const humanizeMessageError = (error: string): string => {
-  let result = error
+export const humanizeMessageError = (error: string): string => {
```

Comment:
Just an enhancement, the proper way to type a variable as a string and add a default value , all in one line, is in this way

```error = ''```

The compiler will understand that as a string, and you have a default value too

Just a link to explorer more on this ...https://www.typescripttutorial.net/typescript-tutorial/typescript-default-parameters/

Reply:
done!